cmd/oras/internal/display/metadata/interface.go

```diff
@@ -30,3 +30,13 @@ type PushHandler interface {
 type AttachHandler interface {
 	OnCompleted(opts *option.Target, root, subject ocispec.Descriptor) error
 }
+
+// PullHandler handles metadata output for pull events.
+type PullHandler interface {
+	// OnLayerSkipped is called when a layer is skipped.
+	OnLayerSkipped()
```

Comment:
nit: should it take a `Descriptor` although the descriptor is not actually used?

Reply:
nit: should it return an error?

Reply:
Added descriptor as parameter. Why an error is needed? Currently there is not output required on layer skipped events.